Finished project

AdamWayne-dev opened the pull request.

AdamWayne-dev committed:
Initial commit

AdamWayne-dev committed:
removed database from files

AdamWayne-dev committed:
Uncommented database initialiser

TheCSharpAcademy reviewed (changes requested):
@AdamWayne-dev Thanks for the submission! At the moment the app crashes when end date is lower than start date. I don't think this is a case for an exception (and definitely not an unhandled one). 

This is a simple validation that can be handled in a less expensive way. I recommend you create a validator method for this and add to your tests 😎

<img width="1642" height="679" alt="image" src="https://github.com/user-attachments/assets/e605e57f-e78b-4ef7-a601-fba8b19828a6" />

AdamWayne-dev committed:
Removed the unhandled exception and embedded the incorrect startdate / enddate error handling into the main UI loop (which now prevents the console crashing).

AdamWayne-dev committed:
Add README for Coding Tracker application

This README provides an overview of the Coding Tracker application, including features, tech stack, setup instructions, usage, and project structure.
